Source/WhistleFactory.swift

```diff
@@ -153,4 +164,11 @@ public class WhistleFactory: UIViewController {
       hide()
     }
   }
+    
+  // MARK: - Gesture methods
+    
+  @objc private func handleTapGestureRecognizer() {
```

Comment:
Yes :) Explained here: http://stackoverflow.com/a/24007718/1093865
Basically without `@objc` it throws an error: `Argument of '#selector' refers to a method that is not exposed to Objective-C`